user_interaction_txt_text.py

```diff
@@ -0,0 +1,94 @@
+import os
+from enum import Enum, auto
+from typing import Optional
+
+from user_file_manager import UserFileManager
+
+
+class MessageScope(Enum):
+    GLOBAL = auto()
+    ACTIVE = auto()
+    PASSIVE = auto()
+
+
+class UserInteraction():
```

Comment:
If we have sevaral UserInteractions, please, add separate folder for all `UserInteraction` classes

Reply:
Need info

Reply:
Done